x-pack/legacy/plugins/code/public/style/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$codeSideBarWidth: 16rem;
+$codeSideBarIndent: 2rem;
```

Comment:
Same here: `$euiSizeXL`

Reply:
@daveyholler in general: what about multipliers larger than 2.5? E.g. should the line above be `16rem` or `calc(#{$euiSize} * 16)` or `calc(#{$euiSizeXL} * 8)`?

Reply:
I'd opt for the former `calc(#{$euiSize} * 16)`